jvb/src/main/java/org/jitsi/videobridge/cc/BitrateController.java

```diff
@@ -686,9 +677,11 @@ private synchronized void update()
             }
         }
 
+        Map<String, VideoConstraints> copyOfVideoConstraintsMap = videoConstraintsMap;
```

Comment:
Did you intend to make a copy of the map here?

Reply:
Just the reference, not a deep copy.

Reply:
The only place you use the "copy" is on the line below. I think it should be removed.

Also the parameters for `allocate`, `prioritize` and `makeEndpointMultiRankList` are named `copyOfVideoConstraintsMap`, but it's not clear what it's a copy of (and indeed it isn't a copy of anything)

Reply:
That's a good point! I've simplified the code a bit.